test/datetime.test.js

```diff
@@ -58,9 +58,8 @@ describe('MySQL datetime handling', function() {
     }).then(function(inst) {
       inst.should.not.eql(null);
       inst.dob.toString().should.eql(d.toString());
-      return done();
     }).catch(function(err) {
-      return done(err);
+      should.not.exist(err);
```

Comment:
Just remove the `catch` statement altogether.

In your solution, `err` stack trace is discarded and the test fails with a new error pointing to this line, which is not helpful.